### PR TITLE
#163317533 Add coveralls to report to the repository

### PR DESCRIPTION
**Pull request by barema4**
### What does this PR do?
This PR displays the percentage of the code that has been tested
### Description of Task to be completed?
- Integrate coveralls with Github repository
- Include coveralls in travis.yml file
- Add bootstrap to design a base UI theme
### How should this be manually tested?
- Install coveralls via `yarn add --dev coveralls`
- Run yarn test --coverage
### What are the relevant pivotal tracker stories
[#163317533](https://www.pivotaltracker.com/story/show/163317533)

### CODE REVIEW

**Review by sekayasin — APPROVED** [review]
@barema4 nice work on adding our new UI theme, LGTM

**Review by alexkayabula — APPROVED** [review]
Looks good to me.

File: src/App.js [hunk 1]
```diff
@@ -1,35 +1,29 @@
...
+import { BrowserRouter as Router, Route } from 'react-router-dom';
 import './App.css';
 import './App.scss';
-import { Provider } from 'react-redux';
 import Navbar from './components/layout/Navbar';
 import Footer from './components/layout/Footer';
 import Landing from './components/layout/Landing';
 import Register from './components/auth/Register';
+import Login from './components/auth/Login';
 import index from './store/index';
```
Comment (RrNn) [inline src/App.js]: Remove this import. `import index from './store/index';` It's no longer necessary

**Review by RrNn — CHANGES_REQUESTED** [review]

**Review by mulondo — APPROVED** [review]
Good job

File: src/App.js (hunk 1 above)
Comment (barema4) [reply src/App.js]: Ok, I have removed it

**Comment by RrNn** [comment]
Line **40**  in `components/auth/Register.js` `this.props.registeruser(newUser, this.props.history);` The method name should be `camelCase`. This crept in our code but you can go ahead and fix it.

**Review by RrNn — CHANGES_REQUESTED** [review]
- The file `Passwordreset.js` should be renamed to `PasswordReset.js`.
- The method name `footer` should be renamed to `Footer`. This too can be fixed along. In `components/layout/Footer.js`

File: public/index.html [hunk 2]
```diff
@@ -4,7 +4,6 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
```
Comment (ekumamait) [inline public/index.html:7]: @barema4  do we need this import?

**Comment by barema4** [comment]
> Line **40** in `components/auth/Register.js` `this.props.registeruser(newUser, this.props.history);` The method name should be `camelCase`. This crept in our code but you can go ahead and fix it.

Have done it

**Comment by barema4** [comment]
> * The file `Passwordreset.js` should be renamed to `PasswordReset.js`.
> * The method name `footer` should be renamed to `Footer`. This too can be fixed along. In `components/layout/Footer.js`

I have also changed the method name from footer to Footer

File: public/index.html (hunk 2 above)
Comment (RrNn) [reply public/index.html:7]: No, we don't need it since we have bootstrap in our dependencies. We removed it

Comment (barema4) [reply public/index.html:7]: @RrNn  I have removed the bootstrapcdn from the index.html

**Review by RrNn — CHANGES_REQUESTED** [review]
Good work, I can see three commits. Squash them into **1** and make one commit with good detail of what the pull request adds or removes.

File: src/__tests__/components/auth/Register.test.js [hunk 3]
```diff
@@ -1,30 +1,42 @@
...
   };
-   const editor = shallow(<Register registeruser={jest.fn()} />);
-  
-  it("should render without crashing", () => {
-    expect(editor).toMatchSnapshot();
+
+  it('test Register component onchange', () => {
+    const wrapper = shallow(<Register {...props} />);
+    const instance = wrapper.instance();
+    console.log('instance ++++++++++++++++', instance);
```
Comment (RrNn) [inline src/__tests__/components/auth/Register.test.js]: Remove this `console.log`

**Review by RrNn — CHANGES_REQUESTED** [review]

File: src/__tests__/components/auth/Register.test.js (hunk 3 above)
Comment (barema4) [reply src/__tests__/components/auth/Register.test.js]: I have removed console.log('instance ++++++++++++++++', instance);

**Review by lindseyme — APPROVED** [review]

**Review by lindseyme — APPROVED** [review]
Good work done but squash the commits to one instead of two

**Review by RrNn — APPROVED** [review]

**Pull request merged**